charts/datadog-operator/values.yaml

```diff
@@ -15,6 +15,9 @@ apiKeyExistingSecret:  # <DATADOG_API_KEY_SECRET>
 # appKey -- Your Datadog APP key
 appKey:  # <DATADOG_APP_KEY>
 
+# DataDog API endpoint, example: https://api.datadoghq.eu
```

Comment:
lets also copy the description of the variable from the `datadog/datadog` chart.

```suggestion
# dd_url -- The host of the Datadog intake server to send Agent data to, only set this option if you need the Agent to send data to a custom URL  
## Overrides the site setting defined in "site".
```